### DISCUSSION TITLE
CitraVR (Beta) v0.5.1

### DISCUSSION THREAD

**Discussion by amwatson** [amwatson/CitraVR#83] [Announcements]
- Added Stats Panel for viewing emulator/VR performance stats
- Added speculative support for Quest 3S (note: in theory, this is unnecessary: Quest 3S should get sorted as "unknown HMD" and be treated like a Quest 2)

<hr /><em>This discussion was created from the release <a href='https://github.com/amwatson/CitraVR/releases/tag/v0.5.1'>CitraVR (Beta) v0.5.1</a>.</em>

**Comment by lorencouse** [comment]
So happy to see this project is still active!  Thank you so much for the hard work.

**Comment by kaktusmisapolak** [comment]
quest 3S should be considered a quest 3, it has the same SoC

**Comment by Emil111337** [comment]
how do you turn off the in game user interface for the bottom screen, whenever i press select on left controller as instructed it just completely turns off the bottom screen.
